Fri, 24 Sep 2021 11:12:07-run_all_case.py-Login_case.py-[line:305]-INFO-[日志信息]: 开始执行脚本test_wrong_pwd：

Fri, 24 Sep 2021 11:12:09-run_all_case.py-Login_case.py-[line:316]-ERROR-[日志信息]: Error! 没有触发等待 5 分钟
